### DISCUSSION TITLE
Adjust RAW settings on a portion of a clip

### DISCUSSION THREAD

**Discussion by jefforig** [latenitefilms/BRAWToolbox#153] [Support]
If we just want to change the RAW settings on a portion of clip, for example, if it was cloudy when we started the take then the sun shines directly on the actor, then the sun goes away.  We tried cutting the BRAW clip with the blade tool in order to change the settings but it restarts the clip at the new cut line.  Hopefully that makes sense.

For further context, these BRAW clips are synchronized clips within a multicam clip because we shot with 3 Pocket 4Ks.  Also, we tried just adjusting the color using the color wheels on the main timeline and the synchronized clip within the multicam clip but it's not the same as adjusting the RAW metadata which fixed the issue.  Only using the colors still clipped the highlights.  Whereas in the RAW metadata it was a quick fix to change the ISO and tons of detail was there.

Just as a heads up we have tried the following, make a new synchronized clip that is an exact copy of the one we want to work with and putting that on a different angle within the multicam clip.  And also as a connected clip on the main timeline.  Both of these are a pretty big hassle.

Also, I just saw in a different discussion that duplicates of synchronized clips are their own instance, so we will duplicate them in the future.  

 In writing this out, I was wondering if we have to use keyframes rather than cutting the BRAW clip?  I am not at the studio at the moment but will try this later. 

Thank you.

**Comment by latenitefilms** [comment]
Yes, keyframes would be the solution to this.
